### DISCUSSION TITLE
v3.5 Developer License

### DISCUSSION THREAD

**Discussion by tomdbarker** [ravendb/ravendb#14944] [Q&A] [answered]
Hi,

How do I acquire and apply a community/developer license for v3.5

Please note: this is for a dev version hosted on my dev PC for a legacy project that needs a small amount of work - upgrading to a newer version of RavenDB is NOT an option.

Thanks :-)

**Comment by ayende** [comment, marked as answer]
I'm afraid that we don't offer community edition for RavenDB 3.5

**Reply by tomdbarker** [reply]
ok thanks, so what license options are available to me for development please?

**Reply by ayende** [reply]
Use developer mode for 3.5.  There is no need for a license (some features will not work, mind).

**Reply by tomdbarker** [reply]
Thx
